docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst

```diff
@@ -802,6 +802,17 @@ Here is an example of this configuration file:
 .. literalinclude:: /../deploy/build-farm-recipes/aws_ec2.yaml
    :language: yaml
 
+``build_farm_tag``
+""""""""""""""""
+
+Use ``build_farm_tag`` to differentiate between different Build Farms in FireSim.
+Having multiple ``config_build.yaml`` files with different ``build_farm_tag``
+values allows you to run many groups of builds at once from the same manager instance.
+The instances launched by the ``buildbitstream`` command will be tagged with
+this value.
```

Comment:
Is the only reason for this tag to externally differentiate between build instances launched by different managers?

I don't think this description added to the docs is accurate---you don't need the tag to run multiple builds at once from a single manager.

For the docs here, I'd just say:
"For CI use only."

Reply:
Correct. I'll add the CI command and be a bit more specific.